app/utils/zip-agenda-files.js

```diff
@@ -21,11 +21,14 @@ async function constructArchiveName(agenda) {
   return `VR_zitting_${formattedDate}_${agendaName}_alle_punten.zip`;
 }
 
-async function fetchArchivingJob(agenda, mandateeIds) {
+async function fetchArchivingJob(agenda, mandateeIds, decisions) {
   let url = `/agendas/${agenda.id}/agendaitems/pieces/files/archive`;
   if (mandateeIds.length) {
     url += '?' + (new URLSearchParams({ mandateeIds }).toString());
   }
+  if (decisions) {
+    url += '?decisions' + (new URLSearchParams({ decisions }).toString());
```

Comment:
I should check before asking, but does this work if you also select ministers? Query params in a HTTP request should be separated by a `&`, only the first query param gets a `?` prefixed. I suspect that this will turn the last mandatee ID in the list into `mandatee-uuid?` instead of `mandatee-uuid`.

Reply:
Yeah just checked if you select ministers/pass `mandateeIds` to this function the service won't actually recognize that you passed the `decisions` query param because it's part of the mandateeIds array `?decisions`.

Try something like:
```js
url += '?' + new URLSearchParams( { mandateeIds, decisions }).toString();
```

This will spit out something like `?mandateeIds=id1,id2,id3&decisions=true`. You can try it in your browser console too. In case `mandateeIds` is an empty list it will be something like `?mandateeIds=&decisions=true` but the service should be able to cope with that.